src/js/game/LevelMVC/LevelModel.js

```diff
@@ -597,9 +597,29 @@ module.exports = class LevelModel {
         return false;
       }
     }
+
+    if (this.checkConflict(this.getMoveDirectionPosition(entity, direction), plane)) {
+      return false;
+    }
     return this.getPlaneToPlaceOn(this.getMoveDirectionPosition(entity, direction), entity) !== null;
   }
 
+  checkConflict(position, placementPlane) {
+    var conflict = false;
+    this.controller.levelEntity.entityMap.forEach ( workingEntity => {
+      if (Position.equals(workingEntity.position, position)) {
+        conflict = true;
+      }
+    });
+
+    if ((this.actionPlane === placementPlane && placementPlane.getBlockAt(position).blockType !== "") ||
+        this.groundDecorationPlane.getBlockAt(position).blockType !== "") {
```

Comment:
I think this case is already covered by [line 596](https://github.com/code-dot-org/craft/pull/440/files#diff-82ea2716b14e187d2ef963445c025e64R596) above?